api/v1alpha3/awsmachine_types.go

```diff
@@ -28,6 +28,17 @@ const (
 	MachineFinalizer = "awsmachine.infrastructure.cluster.x-k8s.io"
 )
 
+// SecretBackend defines variants for backend secret storage.
+type SecretBackend string
+
+var (
+	// SSM defines AWS Systems Manager Parameter Store as the secret backend
+	SSM = SecretBackend("parameter-storer")
+
+	// SecretsManager defines AWS Secrets Manager as the secret backend
+	SecretsManager = SecretBackend("secrets-manager")
```

Comment:
```suggestion
	SecretBackendSecretsManager = SecretBackend("secrets-manager")
```
Makes it easier to see what the var stands for when importing the package